addons/mail/models/mail_channel.py

```diff
@@ -1114,17 +1119,19 @@ def channel_create(self, name, privacy='groups'):
         return channel_info
 
     @api.model
-    def create_group(self, partners_to, default_display_mode=False):
+    def create_group(self, partners_to, default_display_mode=False, name=''):
         """ Create a group channel.
             :param partners_to : list of res.partner ids to add to the conversation
+            :param default_display_mode: string selection that determines how the channel will be displayed by default
+            :param name: string of group name. default name is computed client side from the list of members if no name is set
```

Comment:
The `string` bit is probably unnecessary, or should be moved to the `:param:` e.g.

    :param str default_display_mode: how the channel will be displayed by default